src/angular-business/chip/chip-input/chip-input.component.ts

```diff
@@ -241,6 +273,8 @@ export class ChipInputComponent extends SbbChipsMixinBase
 
   /**
    * Adds a given value to the current selected values.
+   * TODO: Prefix with _
```

Comment:
Wouldn't it be better to already prefix it with "_" and provide a getter and setter to onSelect?